distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc

```diff
@@ -61,4 +61,12 @@ These are the attributes common to any of the media file types which support add
 * `contact.creator-name`
 * `contact.publisher-name`
 * `contact.point-of-contact-phone`
-* `topic.keyword`
\ No newline at end of file
+* `topic.keyword`
+
+Some file types support attributes unique to that individual type:
+
+.Additional Unique Attributes by File Type
+Mp4:: `ext.mp4.audio-sample-rate`
```

Comment:
Yeah I guess it only makes sense if this section will grow with more attributes otherwise it should be singular `attribute`